source/Lib/vvdec/vvdec.cpp

```diff
@@ -141,15 +141,15 @@ int paramCheck( vvdecParams *params )
     return -1;
   }
 
-  if( params->threads > 64 )
+  if( params->threads > MAX_THREADS )
```

Comment:
I wouldn't limit manual setting. Do we even know how many threads are required to cause problems? I think if someone wants to set it higher, its their decision and risk.

Reply:
It was limited like that before, I just applied it for the autodetection also. Sure, we can talk about allowing unlimited threads when setting it manually, but I would like to find out where the deadlock originates first.

Reply:
I see, we had a bug in the parameter check, when threads = -1 would pass this check but later expand to something violating this check. Makes sense